lib/Github/Api/Repo.php

```diff
@@ -196,19 +197,21 @@ public function create(
         $hasWiki = false,
         $hasDownloads = false,
         $teamId = null,
-        $autoInit = false
+        $autoInit = false,
+        $licenseTemplate = 'mit'
```

Comment:
I'm not sure we should use MIT as the default value as github doesn't have a default value. So maybe just an empty string so we don't enforce incorrect default data